roles/confluent.variables/vars/main.yml

```diff
@@ -800,5 +807,11 @@ control_center_combined_properties: "{{control_center_properties | combine_prope
 
 control_center_final_properties: "{{ control_center_combined_properties | combine(control_center_custom_properties) }}"
 
+### The base path for the configuration files. When in Archive File deployment mode this results in configuration files being based in `/opt/confluent/etc`, otherwise they are based in `/etc`.
+config_base_path: "{{ archive_destination_path if installation_method == 'archive' else '' }}"
```

Comment:
Related to the current thread, how about we move this variable back into `roles/confluent.variables/defaults/main.yml` 
That way it can be customized, it cant if it is in this file. And default it to the empty string, and leave a good description of what it can do.

Comment specify also specify that the variable can only be customized if installation_method == 'archive'
Or else we need to test customizing this var for package installation (which i dont think would work)

Reply:
Fixed in commit c340e105.

I defaulted the value to `archive_destination_path`, which is defaulted to `/opt/confluent`. I also updated all references to it to only use it if `installation_method == 'archive'`. This way other installation methods completely ignore it no matter it's value.